packages/cms-lib/api/secrets.js

```diff
@@ -18,7 +18,14 @@ async function deleteSecret(portalId, key) {
   });
 }
 
+async function getSecrets(portalId) {
```

Comment:
Let's rename this to `fetchSecrets`.  Most of our `GET` clients are named `fetch*`.